indico/modules/events/editing/notifications.py

```diff
@@ -0,0 +1,88 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from flask import session
+
+from indico.core.notifications import make_email, send_email
+from indico.modules.events.editing.schemas import EditingConfirmationAction
+from indico.web.flask.templating import get_template_module
+from indico.web.flask.util import url_for
+
+
+def _get_timeline_url(revision):
+    editable = revision.editable
+    return url_for('event_editing.editable', editable, _external=True)
+
+
+def notify_submitter_comment(revision):
+    """
+    Notifies the editor about comments made by a submitter.
```

Comment:
"Notify ..." (PEP257)

Feel free to change the wording in the other docstrings that are wrong, too. We didn't follow this rule initially and only started at some point between 1.9.x and 2.0, so older docstrings are still "wrong"